src/vanilla/Templates/MethodTemplate.cshtml

```diff
@@ -142,12 +142,6 @@ async @(Model.Name.ToCamelCase())WithHttpOperationResponse(@(Model.MethodParamet
   httpRequest.formData = formData;
   </text>      
   }
-  else
-  {
-  <text>
-  httpRequest.body = null;
-  </text>
-  }
```

Comment:
@thecodejedi - This change in the generator should fix your problem. Can you also verify by removing `httpRequest.body = null;` in the generated client for `GET` and `HEAD` methods and test whether it works for you. Make sure that you have reverted your changes in the local copy of `ms-rest-js` while testing this change.

Reply:
@amarzavery Manually removed it after generating the code and restored original ms-rest-js, works like a charm on Firefox 45 now!

I already closed Azure/ms-rest-js#16 as it does fix Azure/ms-rest-js#15.

In which version of autorest will this change be available?

Reply:
`2.0.87` is the latest version of `@microsoft.azure/autorest.typescript`

Reply:
try `autorest --reset` and then execute the command to generate the client. It should bring the latest version while generating the client. Post generation try `autorest --info` to verify that you have the latest version.

Reply:
at anytime if you want to use a preview or a specific version you can say `--use=@microsoft.azure/autorest.typescript@2.0.77`. If you want to use a local build while you are developing the generator you can say `--use=<absolute-path-to/autorest.typescript>`